omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PersonAttributeController1_9Test.java

```diff
@@ -38,10 +41,13 @@ public class PersonAttributeController1_9Test extends MainResourceControllerTest
 	String personUuid = RestTestConstants1_8.PERSON_UUID;
 	
 	private PersonService service;
+
+	private LocationService locationService;
```

Comment:
If you are using it in only one place, you may just directly do Context.getLocationService() from that place.